### PR TITLE
requestStatus flag to optionally disable transport request status

### PR DESCRIPTION
**Pull request by aaroncox**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -8,16 +8,20 @@ export interface BrowserTransportOptions {
     classPrefix?: string
     /** Whether to inject CSS styles in the page header, defaults to true. */
     injectStyles?: boolean
+    /** Whether to display request success and error messages */
```
Comment (jnordberg) [inline src/index.ts:11]: We should say what the default value for this is in the docstring

File: src/index.ts [hunk 2]
```diff
@@ -227,51 +231,61 @@ export default class BrowserTransport implements LinkTransport {
...
+                infoEl.appendChild(infoSubtitle)
+                emptyElement(this.requestEl)
+                this.requestEl.appendChild(logoEl)
+                this.requestEl.appendChild(infoEl)
+                this.show()
+                this.closeTimer = setTimeout(() => {
+                    this.hide()
+                }, 1.5 * 1000)
+            } else {
+                emptyElement(this.requestEl)
```
Comment (jnordberg) [inline src/index.ts:252]: `emptyElement` is only needed before changing the contents

File: src/index.ts [hunk 3]
```diff
@@ -227,51 +231,61 @@ export default class BrowserTransport implements LinkTransport {
...
+                infoEl.appendChild(infoSubtitle)
+                emptyElement(this.requestEl)
+                this.requestEl.appendChild(logoEl)
+                this.requestEl.appendChild(infoEl)
+                this.show()
+                this.closeTimer = setTimeout(() => {
+                    this.hide()
+                }, 5 * 1000)
+            } else {
+                emptyElement(this.requestEl)
```
Comment (jnordberg) [inline src/index.ts:286]: same here

Comment (aaroncox) [reply src/index.ts]: Got it - and changed this and the other one

File: src/index.ts (hunk 1 above)
Comment (aaroncox) [reply src/index.ts]: Added the defaults to the comment!

**Comment by aaroncox** [comment]
Any other feedback or objections to pushing a new build out?

**Pull request merged**